internal/pkg/tekton/pipeline_run_builder.go

```diff
@@ -109,37 +111,52 @@ func NewPipelineRunBuilder(name, namespace string) *PipelineRunBuilder {
 				PipelineSpec: &tektonv1.PipelineSpec{
 					Workspaces: []tektonv1.PipelineWorkspaceDeclaration{
 						{
-							Name: "shared-db",
+							Name: "shared-data",
 						},
 					},
 					Tasks: []tektonv1.PipelineTask{
 						{
 							Name: "build",
 							Workspaces: []tektonv1.WorkspacePipelineTaskBinding{
 								{
-									Name:      "shared-db",
-									Workspace: "shared-db",
+									Name:      "shared-data",
+									Workspace: "shared-data",
 								},
 							},
 							TaskSpec: &tektonv1.EmbeddedTask{
 								TaskSpec: tektonv1.TaskSpec{
 									Workspaces: []tektonv1.WorkspaceDeclaration{
 										{
-											Name: "shared-db",
+											Name: "shared-data",
 										},
 									},
 									Steps: []tektonv1.Step{
 										{
 											Name:   "prepare-db",
 											Image:  "quay.io/konflux-ci/mintmaker-osv-database:latest",
-											Script: "cp -r /data/osv-db /workspace/shared-db",
+											Script: "echo 'Copying OSV database to the shared workspace'; cp -r /data/osv-db /workspace/shared-data",
 											SecurityContext: &corev1.SecurityContext{
 												Capabilities:             &corev1.Capabilities{Drop: []corev1.Capability{"ALL"}},
 												RunAsNonRoot:             ptr.To(true),
+												RunAsUser:                &normalUser,
 												AllowPrivilegeEscalation: ptr.To(false),
-												SeccompProfile: &corev1.SeccompProfile{
-													Type: corev1.SeccompProfileTypeRuntimeDefault,
-												},
+											},
+										},
+										{
+											Name:  "prepare-rpm-cert",
+											Image: "registry.access.redhat.com/ubi9",
+											Script: "[ ! -f \"/etc/renovate/secret/rpm-activationkey\" ] && echo 'RPM secret not found. Exiting.' && exit 0;" +
+												"echo 'Generating RPM certificate and copying it to shared workspace';" +
+												"KEY_NAME=$(cat /etc/renovate/secret/rpm-activationkey);" +
+												"ORG_ID=$(cat /etc/renovate/secret/rpm-org);" +
+												"subscription-manager register --activationkey=\"$KEY_NAME\" --org=\"$ORG_ID\";" +
```

Comment:
We should probably unregister the system from subscription after renovate runs, otherwise there will be one system added to the subscription each time when renovate is executed, which can result in a large amount of registered systems in subscription.

Reply:
I think it should be fine to leave it as is for now, until someone raises a concern or complaint about it :)